src/app/ui/bridge/HalfSizeBlock.tsx

```diff
@@ -49,7 +48,7 @@ function HalfSizeBlock({
   type: 'Blockchain' | 'Token';
 }) {
   return (
-    <BridgePageContainer width={'48.5%'}>
+    <SeoPageContainer width={'48.5%'}>
```

Comment:
same

Reply:
Yep, it´s also named "DynamicPagesContainer" now, because used in multiple pages and not just on the swap pages.